packages/api/src/index.js

```diff
@@ -56,7 +52,11 @@ class Sdk {
 
         const har = oasToHar(spec, operation, data, prepareAuth(authKeys, operation));
 
-        return fetchHar(har, self.userAgent).then(res => {
+        return fetchHar(har, {
+          userAgent: self.userAgent,
+          files: data.files || {},
```

Comment:
Would be interesting to think about how we handle file streams in future: https://nodejs.org/docs/latest/api/fs.html#filehandlecreatereadstreamoptions

I'm also not sure what [this](https://nodejs.org/docs/latest/api/fs.html#filehandlereadablewebstream) is, but it seems like a way to create a ReadableStream that's compatible with the ones that you may find in a browser: https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream/ReadableStream

TIL: https://nodejs.org/docs/latest/api/webstreams.html#overview

Reply:
Yep we support `fs.createReadStream` right now. I'm not sure about `fs.createReadStream`. We'll see!

Reply:
`fs.createReadStream` is Node 16+ only so we can't easily test against it right now.